src/Servers/IIS/IIS/test/testassets/InProcessNewShimWebSite/InProcessNewShimWebSite.csproj

```diff
@@ -59,4 +63,13 @@
     </PackageReference>
     <Reference Include="xunit.assert" />
   </ItemGroup>
+
+  <!-- Repeat Build target for win-x64 to allow later Publish w/ NoBuild=true. -->
```

Comment:
Just confirming that this already covers the Win32/arm64 cases as well, or if I'll need to revisit this section for arm64 later

Reply:
If you add an arm64 profile to the `@(TestAssetPublishProfile)` item group, will need another target here too. Not sure how much the `Condition` will need to change w/o seeing what profile(s) you need to test. Of course, tests might have enough coverage w/o doing the self-contained thing for each possible Windows platform.